src/intervals/interval_operations/boolean.jl

```diff
@@ -0,0 +1,228 @@
+# This file is part of the IntervalArithmetic.jl package; MIT licensed
+
+#=  This file contains the functions described in sections 9.5 of the
+    IEEE Std 1788-2015 (Boolean functions of intervals) and/or required for
+    set-based flavor in section 10.5.9.
+    
+    Some other (non required) related functions are also present, as well as
+    some of the "Recommended operations" (section 10.6.3).
+=#
+
+# Equivalent to `<` but with Inf < Inf being true.
+function isweaklylessprime(a::Real, b::Real)
+    (isinf(a) || isinf(b)) && a == b && return true
+    return a < b
+end
+
+"""
+    ≛(a::Interval, b::Interval)
+
+Checks if the intervals `a` and `b` are identical.
+
+Typed as \\stareq<TAB>.
+
+Implement the `equal` function of the IEEE Std 1788-2015  (Table 9.3).
+
+The more common `==` operator is reserved for flavor dependent pointwise
+equality.
+
+In most case this is equivalent to the built-in `===`.
```

Comment:
I am in favor that we have a pointwise equality operator (which returns `true`, `false` or `missing`), and an equality operator (which returns `true` or `false`) that fulfills the standard; yet, I'm not so sure that these should be `==` and `≛` respectively. In particular, if `≛` is equivalent to `===` I would prefer the latter, which seems to me "more julian"; the question is if there are examples where such equivalence does not hold.

Reply:
The only case so far that I have found where `≛` and `===` are not equivalent is for `DecoratedInterval`s.

Reply:
This is provided by pointwise_boolean.jl (I saw your other comment, this is to document it for potential readers).